sqlglot/expressions.py

```diff
@@ -1133,6 +1133,10 @@ def ctas(self, table, properties=None, dialect=None, parser_opts=None, copy=True
             ),
         )
 
+    @property
+    def named_selects(self):
```

Comment:
Is it strange for this to be on the base Expression instance? Should this be on Select?

Reply:
oh, i didn't catch that. yea that doesn't make sense

Reply:
what about subqueryable?

Reply:
it's on select class

Reply:
doh